src/bundle/Resources/config/routing.yml

```diff
@@ -437,13 +437,28 @@ ibexa.rest.copy_subtree:
         locationPath: "[0-9/]+"
 
 ibexa.rest.move_subtree:
+    alias: ibexa.rest.move_subtree.original
+    deprecated:
+        package: 'ibexa/rest'
+        version: '5.0'
+        message: '"%alias_id%" route is deprecated and will be removed in 6.0. Use "ibexa.rest.move_location" route instead.'
+
+ibexa.rest.move_subtree.original:
```

Comment:
What's the purpose of this alias?

Reply:
It's useful for deprecating old routes (https://symfony.com/blog/new-in-symfony-5-4-route-aliasing#deprecating-routes). Afair we are using it somewhere in the page builder.

Reply:
Specifically here: https://github.com/ibexa/page-builder/blob/main/src/bundle/Resources/config/routing.yaml#L3

Reply:
Removing deprecations for now as agreed